features/dd-sdk-android-rum/api/apiSurface

```diff
@@ -2154,7 +2154,7 @@ data class com.datadog.android.rum.model.VitalEvent
         fun fromJsonObject(com.google.gson.JsonObject): FeatureOperationProperties
     companion object 
       fun fromJson(kotlin.String): Vital
-      fun fromJsonObject(com.google.gson.JsonObject): Vital
+      fun fromJsonObject(com.google.gson.JsonElement): Vital
```

Comment:
we probably need to rename `fromJsonObject` to `fromJsonElement` then, but such change will break Flutter, for example https://github.com/search?q=repo%3ADataDog%2Fdd-sdk-flutter%20fromJsonObject&type=code